front/lib/multi_regions/region_lookup_client.ts

```diff
@@ -0,0 +1,78 @@
+import { EnvironmentConfig, isDevelopment } from "@dust-tt/types";
+
+import type {
+  UserLookupRequestBodyType,
+  UserLookupResponse,
+} from "@app/pages/api/lookup/[resource]";
+
+const CLUSTER_REGIONS = ["local", "europe-west1", "us-central1"] as const;
+type ClusterRegionType = (typeof CLUSTER_REGIONS)[number];
+
+const REGIONS_TO_URLS: Map<ClusterRegionType, string> = new Map([
+  ["local", "http://localhost:3000"],
+  ["europe-west1", "https://eu.dust.tt"],
+  ["us-central1", "https://dust.tt"],
+]);
+
+type Resource = "user";
+
+function isCurrentRegion(region: ClusterRegionType): boolean {
+  return EnvironmentConfig.getEnvVariable("DUST_REGION") === region;
+}
+
+const getAvailableRegions = () => {
```

Comment:
Could we remove `local` from CLUSTER_REGIONS and have this function check whether we are in dev or not?

Reply:
I had to put `local` because it's the DUST_REGION that was set on my dev env (because of KW search I believe).
It feels logical to me to have a region for dev.

I have trouble understanding how it would work for dev environnement without it (what region / url would you return ?). Can you explain how you see it ?

Reply:
I probably didn't think about it enough to see potential problems with it but could we have something like:
```
const getAvailableRegions = () => {
  if (region === "local") {
    return ["http://localhost:3000"]
  }
  return ...
}
```
and take out "local" from `CLUSTER_REGIONS`?

Reply:
I'll keep the "local" region because I think its the more principled approach.